src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -495,38 +524,38 @@ https://cloud.google.com/compute/docs/access/service-accounts[Google Cloud Servi
 Now, you are ready to create a new node source of the type _GCE Infrastructure_.
 The properties needed for the node deployment in the _Create GCE Node Source_ are:
 
--   *gceCredential:* The credentials to perform the basic Google Cloud Platform authentication.
+-   *gceCredential*: The credentials to perform the basic Google Cloud Platform authentication.
```

Comment:
I would have used :*  instead of *:    (i.e. have the colons in bold), but this is only an esthetic choice.

Reply:
I keep the ":" outside to avoid the syntax problem of the property "image". Asciidoc takes `image:` as a diagram. The rest parameters are just to keep uniform.

Reply:
I had the same issue, to fix it I added plus signs around the word "image":
\*Custom Azure +image+:\*